File: test/fixtures/output/php/guzzle6/full.php
<?php

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value', [
    'form_params' => array(
    'foo' => 'bar'
),
    'headers' => [
        'accept' => 'application/json',
        'content-type' => 'application/x-www-form-urlencoded',
        CURLOPT_COOKIE => 'foo=bar; bar=baz',
    ],
]);

echo $response->getBody();

File: src/targets/csharp/restsharp/fixtures/full.cs
var client = new RestClient("http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value");
var request = new RestRequest("", Method.Post);
request.AddHeader("accept", "application/json");
request.AddHeader("content-type", "application/x-www-form-urlencoded");
request.AddCookie("foo", "bar");
request.AddCookie("bar", "baz");
request.AddParameter("application/x-www-form-urlencoded", "foo=bar", ParameterType.RequestBody);
var response = client.Execute(request);